test/github_pullrequest_concurrency_test.go

```diff
@@ -78,27 +77,6 @@ func TestGithubPullRequestConcurrency(t *testing.T) {
 	}
 	assert.NilError(t, wait.UntilMinPRAppeared(ctx, runcnx.Clients, waitOpts, numberOfPipelineRuns))
 
-	allPR, err := runcnx.Clients.Tekton.TektonV1beta1().PipelineRuns(targetNS).List(ctx, metav1.ListOptions{})
```

Comment:
i am lost, wjy is it removed again, or is it just too flaky? :)

Reply:
there is a better check below now so not required!
one in running and one in pending... the check was there when we had 1 concurrency, now doesn't really needed

Reply:
not sure i understood but fine by me :D